kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+
+  <div>
+    <KGrid
+      class="add-padding"
+    >
+      <KGridItem
+        :layout4="{ span: 1 }"
+        :layout8="{ span: 1 }"
+        :layout12="{ span: 1 }"
+      >
+        <KIcon
+          icon="quiz"
+          class="style-icon"
+        />
+      </KGridItem>
+
+      <KGridItem
+        :layout4="{ span: 3 }"
+        :layout8="{ span: 7 }"
+        :layout12="{ span: 11 }"
+      >
+        <KTextbox
+          ref="title"
+          v-model.trim="examTitle"
+          :label="coachString('titleLabel')"
+          :autofocus="true"
+          :maxlength="100"
+        />
+      </KGridItem>
+    </KGrid>
+
+    <p>{{ $tr('addSectionsDescription') }}</p>
+
+    <hr class="bottom-border">
```

Comment:
The color on these line breaks should be [`fineLine`](https://design-system.learningequality.org/colors/#tokens-fineLine)